tests/ts-app-template/app/app.ts

```diff
@@ -1,9 +1,16 @@
 import Application from '@ember/application';
+import compatModules from '@embroider/core/entrypoint';
```

Comment:
Locally I have a TypeScript error `Cannot find module '@embroider/core/entrypoint' or its corresponding type declarations.` but I haven't figured out yet how to declare a virtual module.

Reply:
I wonder if it tries to resolve the `.d.ts` copy of the internal name 🤔 i.e. whatever the virtual content deals with `-embroider-entrypoint.js` is there a resolve request for `-embroider-entrypoint.d.ts` through the same mechanism? 

Does this make sense?

Reply:
i think you just need somewhere
```ts
declare module '@embroider/core/entrypoint' {
  export default ...whatever needs to be here;
}
```

Reply:
or make a `entrypoint.d.ts` in @embroider/core package?

Reply:
maybe it needs to be in global.d.ts ? only if a d.ts file has no imports it becomes a ambient module declarations

Reply:
oh, that also has an import. maybe create a `entry.d.ts` file then?

Reply:
@patricklx that worked, thank you! 🙌